gtda/images/preprocessing.py

```diff
@@ -187,6 +187,11 @@ def plot(Xt, sample=0, colorscale='greys', origin='upper',
 class Inverter(BaseEstimator, TransformerMixin, PlotterMixin):
     """Invert all 2D/3D images in a collection.
 
+    Applies an inversion function to the value of all pixels of all images in
+    the input collection. The inversion function is defined as
+    :math:`f(x) = M - x`, where `x` is a pixel value and `M` is the maximum value
```

Comment:
Is it not more correct to say that `x` is :attr:`max_value_`?

Reply:
You mean `M`, yes xD

Reply:
Yup, thanks. Today is the day I confuse a lot of things with one another!